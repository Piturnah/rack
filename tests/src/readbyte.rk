fn main in
  "test" dup @ print
  dup 1 + @ print
  dup 2 + @ print
  dup 3 + @ print
end
