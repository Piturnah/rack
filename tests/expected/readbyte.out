----STDOUT----
116
101
115
116

----STDERR----
